controllers/postController.js

```diff
@@ -0,0 +1,69 @@
+import * as postService from '../services/postService.js';
+
+export const create = async (req, res) => {
+  try {
+    const { title, content } = req.body;
+    const post = await postService.create({ title, content });
+    res.status(201).json({ data: post });
+  } catch (err) {
+    res.status(500).json({ error: 'add post failed' });
+  }
+};
+
+export const getAll = async (req, res) => {
+  try {
+    const posts = await postService.getAll();
+    res.status(200).json({ data: posts });
+  } catch (err) {
+    res.status(500).json({ error: 'get all posts failed' });
+  }
+};
+
+export const getById = async (req, res) => {
+  try {
+    const post = await postService.getById(req.params.postId);
+    if (!post) {
+      let err = new Error('post not found');
+      err.status = 404;
+      throw (err);
+    }
+    res.status(200).json({ data: post });
+  } catch (err) {
+    res.status(err.status || 500).json({ error: err.message || 'find post by id failed' });
```

Comment:
```suggestion
    res.status(err.status ?? 500).json({ error: err.message ?? 'find post by id failed' });
```

요런식으로 하면 좀 더 깔끔할 거 같아요!

`||`의 경우 falsy한 값을 대입했을 떄 오른쪽거를 리턴시켜주는데 falsy한 값이 올일은 없겠지만 그보단 nullable한 값이 왔을 때 (`err.status`가 없거나 `err.message`가 없을 때) 대체할 텍스트나 status code를 리턴하는게 좀 더 깔끔할거 같아요!